cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,31 @@
 describe('', () => {
```

Comment:
The describe function should have a description. Add a brief description about what this group of tests are about.